reference-architecture/aws-ansible/README.md

```diff
@@ -56,24 +69,45 @@ export AWS_SECRET_ACCESS_KEY=bar
 The default region is us-east-1 but can be changed when running the ose-on-aws script by specifying --region=us-west-2 for example. The region must contain at least 3 Availability Zones. 
 
 ### AMI ID
-The AMI ID may need to change if the AWS IAM account does not have access to the Red Hat Cloud Access gold image or if deploying outside of the us-east-1 region.
+The AMI ID may need to change if the AWS IAM account does not have access to the Red Hat Cloud Access gold image, deploying CentOS, or if deploying outside of the us-east-1 region.
 
 ### Containerized Installation
 Specifying the configuration trigger --containerized=true will install and run OpenShift services in containers. Both Atomic Host and RHEL can run OpenShift in containers. When using Atomic Host the version of docker must be 1.10 or greater and the configuration trigger --containerized=true must be used or OpenShift will not operate as expected.
 
 ### New AWS Environment (Greenfield)
 When installing into an new AWS environment perform the following.   This will create the SSH key, bastion host, and VPC for the new environment.
+
+**OpenShift Container Platform**
 ```
 ./ose-on-aws.py --keypair=OSE-key --create-key=yes --key-path=/path/to/ssh/key.pub --rhsm-user=rh-user --rhsm-password=password --public-hosted-zone=sysdeseng.com --rhsm-pool="Red Hat OpenShift Container Platform, Standard, 2-Core"
 ```
+**OpenShift Origin**
+```
+./ose-on-aws.py --keypair=OSE-key --create-key=yes --key-path=/path/to/ssh/key.pub --public-hosted-zone=sysdeseng.com --deployment-type=origin --ami=ami-6d1c2007 
+```
 
 If the SSH key that you plan on using in AWS already exists then perform the following.
+
+**OpenShift Container Platform**
 ```
 ./ose-on-aws.py --keypair=OSE-key --rhsm-user=rh-user --rhsm-password=password --public-hosted-zone=sysdeseng.com --rhsm-pool="Red Hat OpenShift Container Platform, Standard, 2-Core"
 
 ```
+
+**OpenShift Origin**
+```
+./ose-on-aws.py --keypair=OSE-key --public-hosted-zone=sysdeseng.com --deployment-type=origin --ami=ami-6d1c2007
+```
+
 ### Existing AWS Environment (Brownfield)
-If the installing OpenShift into an existing AWS VPC perform the following. The script will prompt for vpc and subnet IDs.  The Brownfield deployment can also skip the creation of a Bastion server if one already exists. For mappings of security groups make sure the bastion security group is named bastion-sg.
+If the installing OpenShift Container Platform or OpenShift Origin into an existing AWS VPC perform the following. The script will prompt for vpc and subnet IDs.  The Brownfield deployment can also skip the creation of a Bastion server if one already exists. For mappings of security groups make sure the bastion security group is named bastion-sg.
```

Comment:
Typo:
If **the** installing..

Missing Comma:
an existing AWS VPC, perform 

**A** Brownfield deployment can also

For mappings of security groups, make